tests/platform_tests/api/test_watchdog.py

```diff
@@ -151,34 +174,13 @@ def test_arm_too_big_timeout(self, duthost, platform_api_conn, conf):
         if watchdog_timeout is None:
             pytest.skip('"too_big_timeout" parameter is required for this test case')
         actual_timeout = watchdog.arm(platform_api_conn, watchdog_timeout)
-
-        assert actual_timeout == -1
+        self.expect(actual_timeout == -1, "{}: watchdog time {} shouldn't be set".format(test_arm_too_big_timeout.__name__, watchdog_timeout))
```

Comment:
Suggest changing message to,

```
"{}: Watchdog should be disarmed, but returned timeout of {} seconds".format(test_arm_too_big_timeout.__name__, watchdog_timeout))
```

Reply:
Please use my suggested wording here. It is clear and simple, and the test name will get prefixed to the error message, so we will know which test failed with this message.